packages/core/src/parser/ParserUtil.ts

```diff
@@ -17,6 +17,7 @@ export const basicSymbols: moo.Rules = {
   rparen: ")",
   apos: "'",
   comma: ",",
+  hex_literal: /#(?:[0-9A-Fa-f]{3,})/,
```

Comment:
Why do you use `(?:)` here?

Reply:
Probably from a previous attempt at capturing just the hex string. Removing now.